proto/regen/ecocredit/v1alpha1/tx.proto

```diff
@@ -39,6 +40,18 @@ service Msg {
 
   // UpdateClassMetadata updates the credit class metadata
   rpc UpdateClassMetadata(MsgUpdateClassMetadata) returns (MsgUpdateClassMetadataResponse);
+
+  // Sell creates new sell orders.
+  rpc Sell(MsgSell) returns (MsgSellResponse);
+
+  // UpdateSellOrders updates existing sell orders.
+  rpc UpdateSellOrders(MsgUpdateSellOrders) returns (MsgUpdateSellOrdersResponse);
+
+  // BuyDirect creates buy orders directly against sell orders.
+  rpc BuyDirect (MsgBuyDirect) returns (MsgBuyDirectResponse);
+
+  // AllowAskDenom is a governance operation which authorizes a new ask denom to be used in sell orders
+  rpc AllowAskDenom (MsgAllowAskDenom) returns (MsgAllowAskDenomResponse);
```

Comment:
so this is relying on https://github.com/cosmos/cosmos-sdk/issues/9438?

Reply:
Yes